integration-tests/run_cluster.sh

```diff
@@ -47,12 +47,25 @@
   # Make directories if they dont exist
   mkdir -p $SHARED_DIR/logs
   mkdir -p $SHARED_DIR/tasklogs
+  mkdir -p $SHARED_DIR/docker/extensions
+  mkdir -p $SHARED_DIR/docker/credentials
 
   # install druid jars
   rm -rf $SHARED_DIR/docker
   cp -R docker $SHARED_DIR/docker
   mvn -B dependency:copy-dependencies -DoutputDirectory=$SHARED_DIR/docker/lib
 
+  # move extensions into a seperate extension folder
+  # For druid-s3-extensions
+  mkdir -p $SHARED_DIR/docker/extensions/druid-s3-extensions
```

Comment:
hmm, this doesn't seem a great way to handle this and doesn't really reflect how this will likely be run in practice, because the extension dependency jars will still all be on the classpath, just the extension jar itself will be in another folder. I think we will probably want to get the druid install similar to how it is in normal packaging to provide a more realistic test environment, which probably precludes using `mvn -B dependency:copy-dependencies`, or at least the way we are currently using it, so that way all extensions are split out in the separate extensions folder and not on the classpath.

I guess this could be fine for now though, and we could figure out how to do this in a future PR.

Reply:
I agree. It's very hard to separate out the extensions cleanly using `mvn -B dependency:copy-dependencies`
Moving away from `mvn -B dependency:copy-dependencies` to how it is in normal packaging using the distribution is the right way in my opinion but is a much bigger/unrelated change than the purpose of this PR. For the purpose of this PR, as long as we don't load the extension jar which checks for configs/credentials to be set, we are good.